frontend/packages/core/src/Table/table.tsx

```diff
@@ -96,6 +96,8 @@ const TableContainer = ({ children }: TableContainerProps) => (
 export interface TableProps extends Pick<MuiTableProps, "stickyHeader"> {
   /** The names of the columns. This must be set (even to empty string) to render the table. */
   columns: string[];
+  /** The breakpoint at which to compress the table rows. By default the small breakpoint is used. */
+  compressBreakpoint?: "xs" | "sm" | "md" | "lg" | "xl";
```

Comment:
Can we use material's interface for these?
`import type { BreakpointDefaults } from "@material-ui/core/styles/createBreakpoints";`